README.md

```diff
@@ -447,33 +445,51 @@ In some cases, the latest features or bug fixes may be integrated into the main
 
 <sub>**Warning:** _Features and bug fixes that are implemented on the latest version of Terrain may still be subject to testing. As such, you should only use the main branch of the Terrain github repo in exceptional circumstances. In all other cases, use the npm package._</sub>
 
-To use the main branch of the Terrain repo on your local machine, do the following:
+To use the main branch of the Terrain repo on your local machine, follow the procedure below.
 
-1. Clone the repo.
+1. Ensure the terrain package has been installed globally.
+
+```sh
+npm install -g @terra-money/terrain
+```
```

Comment:
This step is not needed to work on Terrain itself! Curious why you added it?

Reply:
@octalmage On step 5 (4 in the current README) it says to run npm link to link the local terrain repo to your global terrain instance.  So I just thought it would make more sense to start with how to install terrain globally.